### PR TITLE
HIPAA pricing: increase customer comfort

### LINKED ISSUE #478: HIPAA pricing page: increase customer comfort
Consider adding to the pricing page a few things to make customers more comfortable, including “need help deciding,” the “no risk,” “no lock-in,” “no gruntwork” badges, and an FAQ. See the bottom of the Heroku pricing page for reference.

### PR DESCRIPTION
**Pull request by zackproser**
Fixes #478

### CODE REVIEW

File: pages/landing/hipaa/template/pricing/index.html [hunk 1]
```diff
@@ -29,24 +29,79 @@
...
-      - name: HIPAA-compliant application templates
-        description: World-class application templates that you can easily extend.
+      - name: HIPAA-compliant sample applications
+        description: World-class sample applications that you can easily extend.
       - name: Infrastructure as Code Library
         description: Access our entire library to augment and extend your reference architecture.
       - name: Gruntwork customer support
         description: Gruntwork engineers can answer your technical questions and help you resolve problems if they come up.
-      - name: Access to our library of training videos to ramp up on tools and techniques
+      - name: Access to our library of training videos to ramp up on tools and techniques.
```
Comment (pete0emerson) [inline pages/landing/hipaa/template/pricing/index.html:38]: This is inconsistent with the rest. `name:` has no period, `description` does.

File: pages/landing/hipaa/template/pricing/index.html [hunk 2]
```diff
@@ -29,24 +29,79 @@
...
+faq:
+  - question: What sort of AWS bill should I expect?
+    answer: |
+      <p>
+        Your AWS bill depends on a number of factors: which services you use, the instance types you pick, how much load you're dealing with, etc. That said, assuming you have access to the AWS Free Tier, here's a ballpark range for what your AWS bill will be if you deploy an app using infrastructure from the following subscription option:
+      </p>
+      <ul>
+        <li><strong>Gruntwork Lite</strong>: $0 / month</li>
+        <li><strong>Gruntwork Pro</strong>: $500 / month and up</li>
+        <li><strong>Gruntwork Pro</strong>: $2,000 / month and up</li>
```
Comment (pete0emerson) [inline pages/landing/hipaa/template/pricing/index.html:56]: Should this be Gruntwork Enterprise?

**Review by pete0emerson — COMMENTED** [review]
Made two change requests.

File: pages/landing/hipaa/template/pricing/index.html (hunk 2 above)
Comment (zackproser) [reply pages/landing/hipaa/template/pricing/index.html]: Good catch - I lifted this from the heroku page's pricing FAQ - so we may need to update it there as well - fixed in dde5ca1a

File: pages/landing/hipaa/template/pricing/index.html (hunk 1 above)
Comment (zackproser) [reply pages/landing/hipaa/template/pricing/index.html]: Eagle eyes - fixed in dde5ca1a

**Comment by zackproser** [comment]
@pete0emerson Thanks for the review! I've addressed your feedback.

**Review by pete0emerson — APPROVED** [review]

**Comment by zackproser** [comment]
Thanks for the review! Going to merge this in now.

**Pull request merged**